deploy/helm/quarks-job/templates/operator.yaml

```diff
@@ -24,6 +24,16 @@ spec:
           - quarks-job
           imagePullPolicy: {{ .Values.global.image.pullPolicy | quote }}
           env:
+            - name: APPLY_CRD
+              value: "{{ .Values.applyCRD }}"
+            {{- if .Values.kubeconfig }}
+            - name: KUBECONFIG
```

Comment:
I don't think this should be part of the helm chart, as it's not needed for running the operator in-cluster?

Reply:
Yes. Removed.